Observer/Admin/DashboardInstantiation.php

```diff
@@ -127,15 +127,17 @@ public function execute(Observer $observer) : void
             $moduleVersion = $moduleData['setup_version'] ?? '';
             $storeEmail = $this->scopeConfig->getValue('trans_email/ident_sales/email') ?? '';
             $storeUrl = rtrim($this->storeManager->getStore()->getBaseUrl(UrlInterface::URL_TYPE_WEB, true), '/');
+            $merchantId = $this->config->getMerchantId();
+            $merchantId = empty($merchantId) ? $this->config->generateMerchantId() : $merchantId;
```

Comment:
Maybe this could be simplified like:
```
$merchantId = $this->config->getMerchantId() ?: $this->config->generateMerchantId();
```

Reply:
@davehensleyns8 It does not unfortunately although I had a similar attempt in addition to:
```
$merchantId = $this->config->getMerchantId() ?: $this->config->generateMerchantId();
```